tests/unit_tests/models/test_model_bridge_lora.py

```diff
@@ -375,6 +392,253 @@ def megatron_to_hf(self, weight, module):
     assert torch.all(materials[0].linear_out_weight.weight == 2 * torch.ones(2, 2))
 
 
+def test_stream_adapter_weights_megatron_to_hf(monkeypatch):
+    bridge = DummyBridge()
+
+    adapter_task = AdapterWeightConversionTask(
+        global_base_prefix="decoder.layers.0.mlp.linear_fc1",
+        adapter_key=None,
+        alpha=2,
+        dim=4,
+        linear_in_task=WeightConversionTask(
+            param_name="local_in",
+            global_param_name="decoder.layers.0.mlp.linear_fc1.adapter.linear_in.weight",
+            mapping=Mock(),
+        ),
+        linear_out_task=WeightConversionTask(
+            param_name="local_out",
+            global_param_name="decoder.layers.0.mlp.linear_fc1.adapter.linear_out.weight",
+            mapping=Mock(),
+        ),
+    )
+
+    adapter_weight = AdapterWeight(
+        global_base_prefix="decoder.layers.0.mlp.linear_fc1",
+        adapter_key=None,
+        alpha=2,
+        dim=4,
+        linear_in_weight=MegatronWeightTuple("local_in", torch.ones(2, 2), vp_stage=0),
+        linear_out_weight=MegatronWeightTuple("local_out", 2 * torch.ones(2, 2), vp_stage=0),
+    )
+
+    monkeypatch.setattr(
+        bridge,
+        "build_adapter_conversion_tasks",
+        lambda *_: {"decoder.layers.0.mlp.linear_fc1": [adapter_task]},
+    )
+    monkeypatch.setattr(
+        bridge,
+        "materialize_adapter_weights",
+        lambda *_: [adapter_weight],
+    )
+
```

Comment:
To fix the test cases:

```suggestion

    # Provide a base HF weight name so stream_adapter_weights_megatron_to_hf can
    # translate it into lora_A/lora_B names.
    monkeypatch.setattr(
        bridge,
        "_get_base_hf_weight_names_for_adapter",
        lambda *_args, **_kwargs: ["model.layers.0.mlp.linear_fc1.weight"],
    )

```